admin/cli.py

```diff
@@ -3,7 +3,7 @@
 import random
 import logging
 
-from os import getenv
+from os import getenv, path, makedirs, walk
```

Comment:
I noticed we imported specific things from `os` instead of just importing the whole thing, I assumed that was for a reason and kept up the practice